### PR TITLE
[25] Add bell sound packs (ZTM Transit bell / more)

### PR DESCRIPTION
**Pull request by Mateusz-Dobrzynski**
Implements #25. This PR implements support for ZTM Poznań pack. Other sound packs can be added by expanding `types/soundPack.ts`. Submitting custom sound packs to be done later.

### CODE REVIEW

File: src/app/oxford-debate/page.tsx [hunk 1]
```diff
@@ -76,11 +91,7 @@ export default function OxfordDebate() {
             {/* {debate.conf.proTeam ? "as the proposition" : "in favour"} */}
             {aspropo}
           </p>
-          <Dots
-            stages={[0, 2, 4, 6]}
-            stage={stage}
-            flashCurrent={running && !advocem}
-          />
+          <Dots stages={[0, 2, 4, 6]} stage={stage} flashCurrent={running && !advocem} />
```
Comment (jakubmanczak) [inline src/app/oxford-debate/page.tsx:94]: This is aimless, probably due to a code formatter, right? I feel like the prior version was more readable.

File: src/app/oxford-debate/page.tsx [hunk 2]
```diff
@@ -10,12 +10,9 @@ import { IconStopCircle } from "@/components/icons/StopCircle";
...
 import { IconArrowLeftCircle } from "@/components/icons/ArrowLeftCircle";
 import Link from "next/link";
+import { useAudio } from "react-use";
 
-const Dots = (props: {
-  stages: number[];
-  flashCurrent: boolean;
-  stage: number;
-}) => {
+const Dots = (props: { stages: number[]; flashCurrent: boolean; stage: number }) => {
```
Comment (jakubmanczak) [inline src/app/oxford-debate/page.tsx:15]: This makes things more difficult to read - too many things going on.

**Review by jakubmanczak — CHANGES_REQUESTED** [review]

**Comment by Mateusz-Dobrzynski** [comment]
Fixed. We probably should unify configs of our code formatters to avoid such hiccups in the future.

**Comment by jakubmanczak** [comment]
I think adding a preview of these new sounds to the Sound Demonstration page would be nice, as well as a file in the public directory about their origin, to supplement our license.

Something like

"LICENSE-NOTE"
```
The following files in this directory are sourced from ...
under the following terms ...
and are not subject to the distribution terms in the general LICENSE file...
Whatever else... Blah blah ... Don't sue us
```

**Comment by jakubmanczak** [comment]
There's also still a lot of these aimless changes which you can observe in the 'Files Changed' diff that mess up the git blame.

**Comment by Mateusz-Dobrzynski** [comment]
I think a `LICENSE-NOTE`, is an awkward solution (especially that the name is not quite descriptive). ZTM sounds are not our only dependencies. It's the only one that requires attribution (for now) but I think could devise a future-proof solution, should we add more attribution-required dependencies. On a related note, the motions from the Generator could use an attribution, too.

**Review by jakubmanczak — APPROVED** [review]

**Pull request merged**